docs/OBOOpsLogistics.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: doc
+id: OBOOpsLogistics
```

Comment:
Does the ID do anything? I couldn’t find any places these are used.

Reply:
No idea! I copied the format from another page.

Reply:
They don’t do anything. I can go through and remove them all so it’s one less thing people have to think about to maintain the site

Reply:
They dont somehow relate to the http URLs?

Reply:
No. the URL is generated based on the filename, which is really terrible. A better way with jekyll is to assign a permalink (see https://jekyllrb.com/docs/permalinks/)

I was actually thinking about asking if we could do this, since it would allow us to rearrange more of the site to make sense

Reply:
ok, interesting! Is this though the right thing to focus our limited time on? I don't mind, whatever you think!

Reply:
if you think there are other reasons nobody (besides Nomi and Darren) maintains the OBO site besides its immense complexity and lack of organization, I'm all ears

Reply:
lack of interest

Reply:
Nico is right. It's not like there are people out there thinking, "I am all primed to improve the OBO website! Oh, wait, no, it has these IDs I don't understand; never mind."

But if you want to spend your time simplifying it, Charlie, that's fine! I somewhat disagree, though, with your "don't repeat information in more than one place" philosophy here. I think sometimes it's useful to repeat information in multiple places that people might look (though of course that has to be weighed against the hassle of keeping it up to date in multiple places).